livefs_edit/actions.py

```diff
@@ -380,15 +380,15 @@ def add_debs_to_pool(ctxt, debs: List[str] = ()):
         new_packages.write(cp.stdout)
     with gzip.open(packages + '.gz', 'wb') as new_packages:
         new_packages.write(cp.stdout)
-    release = ctxt.p('new/iso/dists/stable/Release')
+    release = ctxt.p(f'new/iso/dists/{dist}/Release')
     with open(release) as o:
         old = deb822.Deb822(o)
     for p in release, release + '.gpg':
         rm_f(p)
     cp = run(
         [
             'apt-ftparchive', '--md5=off', '--sha1=off', '--sha512=off',
-            'release', 'dists/stable',
+            'release', 'dists/'+dist+'',
```

Comment:
Can you make this an f-string?

Reply:
Done here https://github.com/mwhudson/livefs-editor/pull/19/commits/971c140580253de53d0b136e27d2f6927ad7357d#diff-7e712e896e047ec5c88c221ef19ce9e444b4bf5cf36c0d588b00ef952e1b04faR391